lib/wibox/init.lua

```diff
@@ -81,7 +81,8 @@ function wibox:to_widget()
         shape              = self._shape,
         forced_width       = self:geometry().width  + 2*bw,
         forced_height      = self:geometry().height + 2*bw,
-        widget             = wibox.container.background
+        widget             = wibox.container.background,
+        input_passthrough  = false
```

Comment:
shouldn't it be `self.input_passthrough` instead of `false`?

Reply:
I'm not exactly sure, I'm still trying to understand better how the entire codebase works. I thought those were default values and I just went by the documentation and set that to false. I tried changing it with what you suggested and it still works.

Reply:
I'd say that this is completely unrelated. `:to_widget()` is seldom used.

Can you get rid of the change above? The other change in here looks fine. Also, thanks a lot for figuring this out and thanks for sending a PR to fix this.

Reply:
Yeah, no problem! I changed it back!